### ISSUE TITLE
Trebuchet launcher is not removed via gapps-config

### ISSUE THREAD

**Issue by GermanKiwi** [opengapps/opengapps#191]
I"m using a gapps-config.txt file to remove apps that I don't want to have. I've included the `Launcher` keyword, which I assume should remove the stock launcher. However it doesn't remove the Tebuchet launcher which is included with CM12.

The path to this launcher app is:
/system/priv-app/Trebuchet/Trebuchet.apk

Could you please include it in the `Launcher` keyword, or alternatively create a new keyword for removing it?

**Comment by mfonville** [comment]
If you use ``Launcher`` priv-app/Trebuchet/ is correctly removed.
The installer just won't let you have not NO launcher installed, unless you specify ``override`` too.
Please check the documentation for details, and use XDA for questions like this instead of directly filing an issue.

I closed all 3 issues of you just now, for some things that could easily be found in documentation, previous bugs or asked on XDA :-/

**Comment by FBachofner** [comment]
I have been having the same problem on LineageOS 17.1 and now 18.

Considerations:
1. I *DO* have another launcher installed (Nova) and it is set to be the default launcher
2. I have called "override" in .gapps-config as required (I have tried it both with and without a capital "O" as there are ambiguous examples regarding this.
3. Whether I specify "Launcher" to be removed in the `### EXAMPLE INCLUDE INSTALL START ###` section or whether I specify "(Launcher)" in the section where one can `enclose the name of the apk in parentheses`

Trebuchet simply will not remove!

I have also used the following keywords:
`
(Launcher3)                    # attempt to remove by different APK name
(launcher3)                     # change case?!
(Launcher)                      # attempt to remove by yet ANOTHER different APK name
(Trebuchet)                     # attempt to remove by different APK name
(trebuchet)                      # change case?!
`
None of these options serve to remove Trebuchet.

Any input is appreciated.  Thanks in advance.
